Quantum-Control-Applications-QuAM/Superconducting/quam_libs/components/tunable_coupler.py

```diff
@@ -0,0 +1,35 @@
+from quam.components import SingleChannel
+from quam.components.ports import LFFEMAnalogOutputPort
+from quam.core import quam_dataclass
+
+__all__ = ["TunableCoupler"]
+
+
+@quam_dataclass
+class TunableCoupler(SingleChannel):
+    """
+    Example QuAM component for a tunable coupler.
+
+    Args:
+        decouple_offset (float): the tunable coupler for which the .
+        interaction_offset (float): the tunable coupler for which the .
+    """
+
+    output_mode: str = "amplified"
+    upsampling_mode: str = "pulse"
+
+    decouple_offset: float = 0.0
+    interaction_offset: float = 0.0
+
+    def __post_init__(self):
+        if isinstance(self.opx_output, LFFEMAnalogOutputPort):
+            self.opx_output.upsampling_mode = self.upsampling_mode
+            self.opx_output.output_mode = self.output_mode
+
```

Comment:
@nulinspiratie seems reasonable to me that the coupler inherets properties from the nerighboor flux channels actually since its very likely the behaviors will replicate. Either they are 1e9 or 2e9 or mw or pulsed --

Reply:
@KevinAVR in this case it's not inheriting from the neighbouring flux channels. Instead, the coupler channel is taking these properties from its port. And this behaviour leads to issues.